Source/Scene/Cesium3DTileBatchTableResources.js

```diff
@@ -282,27 +297,116 @@ define([
             result);
     };
 
+    // TODO : put this helper somewhere else?
+    function getComponentCountFromType(type) {
```

Comment:
Perhaps here and below are similar to [getModelAccessor.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/getModelAccessor.js)?  That is private so it can be updated/renamed and even cherry-picked into master early.